src/pages/aens/AuctionBid.vue

```diff
@@ -1,5 +1,6 @@
 <template>
-  <MobilePage
+  <component
+    :is="$globals.IS_MOBILE_DEVICE ? 'MobilePage' : 'DesktopPage'"
```

Comment:
Proposing to replace this with a general Page component (that should look well enough on both desktop and mobile)